### PR TITLE
Complete MVP

### PR DESCRIPTION
**Pull request by RachelGreenwood**
Project Objective: Melody Minded is a music theory education web application to teach children ages 5-15 music theory
Checklist: https://docs.google.com/document/d/1VqNX4G55fdgDmJxw7H68TEHguvbAlvXcUlq1Ld7ACt8/edit?usp=sharing and https://docs.google.com/document/d/1BCmUnw03qKaln2GiESZe_VsINxtbftnrVdi6B5ntsTU/edit?usp=sharing
Done:
-Styling
-Fixed bugs (repeating header, repeating lesson 1 in navbar, next lesson not resetting answers)
-Checked accessibility
-Added button to next lesson (or to forum, if it's the last lesson)
-Concepts and questions can now have images and paragraph breaks
-The text-to-speech API ignores the images and paragraph breaks

To Do:
-Re-deploy to Render
-Add integration tests
-Add content instead of placeholder text
-Clean up code
-Hopefully start forum feature

### CODE REVIEW

**Comment by Technically-Teta** [comment]
@RachelGreenwood this look amazing! I like the way you laid out all the corrections and updates you made as well as styling methods you took. I feel that this gives great insight into your process as an engineer. Well done! I don't see any areas that need additional info. Looks like you're ready to go!

File: client/src/App.test.js [hunk 1]
```diff
@@ -1,73 +1,37 @@
...
-      }}
-      />
-    );
-
-  const button = getByText('correct');
-  button.click();
-  expect(button).toHaveClass('selected')
-  expect(button.parentElement.querySelector('.feedback')).toHaveTextContent('nice work')
+import { BrowserRouter } from 'react-router-dom';
+
```
Comment (islador) [inline client/src/App.test.js:6]: Loving seeing the tests making an appearance! Great solid progress!

File: client/src/components/Home Page/navbar.js [hunk 2]
```diff
@@ -2,40 +2,42 @@ import React from 'react';
 import { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom';
 import Logo from '/Users/tpl1122_2/ProjectCRUDTemplate/client/src/Logo.png';
+import './navbar.css';
```
Comment (islador) [inline client/src/components/Home Page/navbar.js:6]: Component filenames shouldn't have spaces in them.

File: client/src/components/Lesson Page/lesson.js [hunk 3]
```diff
@@ -4,25 +4,38 @@ import Section from './section.js';
...
+        fetch(`/lessons_new/${lessonId}`)
+        .then((response) => response.json())
+        .then((lessons_new) => {
+            setSection(lessons_new);
+        });
+    }
+
+    // Fetches all lessons from /lessons_new table (for Next component)
+    const loadAllLessons= () => {
+        fetch(`/lessons_new/`)
```
Comment (islador) [inline client/src/components/Lesson Page/lesson.js:29]: copy paste error? why is this called `lessons_new`? Seems off.

File: client/src/components/Lesson Page/section.js [hunk 4]
```diff
@@ -11,6 +12,14 @@ const Section = (props) => {
...
   // Logs if chosen answer is correct
   const [isCorrect, setIsCorrect] = useState(null);
+  // Formats concepts to have paragraph breaks
+  const bodyWithLineBreaks = lesson.concept.replace(/\\n/g, '\n');
+  const paragraphs = bodyWithLineBreaks.split('\n');
+
+  // Formats text to not send images to text-to-audio API
+  const conceptToAudio = paragraphs.join('').replace(/<img[^>]*>/g);
+  const questionToAudio = lesson.question.replace(/<img[^>]*>/g);
+
```
Comment (islador) [inline client/src/components/Lesson Page/section.js:22]: The need for all these replacements strikes me as a code smell; the backend should be smart enough to send the frontend what it needs.

That said, not a major issue.

**Review by islador — APPROVED** [review]
Minor nits at this point. I'll get it running locally next round.

That said; each component should be a PR, not this monster PR. Small PRs make everyone happy and get actual feedback.

**Pull request merged**